commands/profile.js

```diff
@@ -19,7 +19,7 @@ const getProfile = async (
             await userTable
                 .select({
                     maxRecords: 1,
-                    filterByFormula: `{discordId} = "${targetUsername}"`,
+                    filterByFormula: `{discordUsername} = "${targetUsername}"`,
```

Comment:
Ah we actually want this to still searh by discord id here. This is because the username could change but the id will always be the same. Sorry if that wasn't clear in the description

Reply:
Ok. Só the search will be by the Id, and only in the answers , we will see the username. For example: 

Command: /profile 4bxb5bxbs3bxnudb

Returns: D3ny5's profile...